README.md

```diff
@@ -40,6 +40,23 @@ jobs:
         filePath: "_manifest/spdx_2.2/"
 ```
 
+Add support for running inside a matrix by overriding the default correlater unique identifier to include the job+matrix values.  Consider these sample steps:
```

Comment:
Sorry to ping like this, is this PR waiting for anything in particular that I can help out with?

A repo I work with creates multiple artifacts via matrixes which each have individual SBOMs, and it would be great to use the functionality in this PR to submit the dependencies for each of these artifacts.

If it requires additional work, I'd be happy to take a look if you'd like.

Reply:
Thanks for the ping was awaiting reviews